### PR TITLE
feat(lang): add daemon function to run daemon process in the container

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingyang@tensorchord.ai>

* related to #769

### CODE REVIEW

File: pkg/builder/util.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func ImageConfigStr(labels map[string]string,
 // DefaultPathEnvUnix is unix style list of directories to search for
 // executables. Each directory is separated from the next by a colon
 // ':' character .
-const DefaultPathEnvUnix = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/conda/bin:/usr/local/julia/bin"
+const DefaultPathEnvUnix = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/conda/bin:/opt/conda/envs/envd/bin/:/usr/local/julia/bin"
```
Comment (kemingy) [inline pkg/builder/util.go:66]: `bash -c` doesn't work. I guess it's because the conda env is added by `source /opt/conda/bin/activate envd`

Comment (kemingy) [reply pkg/builder/util.go:66]: Any suggestions? @gaocegege

Comment (gaocegege) [reply pkg/builder/util.go:66]: LGTM

File: pkg/lang/ir/types.go [hunk 2]
```diff
@@ -55,6 +55,7 @@ type Graph struct {
 	Mount           []MountInfo
 	Entrypoint      []string
 	RuntimeCommands map[string]string
+	RuntimeDaemon   []string
```
Comment (gaocegege) [inline pkg/lang/ir/types.go:58]: Should it be `[][]string`?

Users may define more than one daemon process.

Comment (kemingy) [reply pkg/lang/ir/types.go:58]:
```docker
# 1
[["date", "-u"], ["sleep", "infinity"]]
# 2
["date -u", "sleep infinity"]
```
Which one do you prefer?

Comment (gaocegege) [reply pkg/lang/ir/types.go:58]: 1 is better for me.

Comment (kemingy) [reply pkg/lang/ir/types.go]: Fixed. PTAL.

**Review by aseaday — APPROVED** [review]
LGTM

**Review by VoVAllen — APPROVED** [review]
LGTM

**Pull request merged**